<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tab_Map">
        <attribute name="title">
         <string>Map</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QGraphicsView" name="mapGraphicsView"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_Locomotion">
        <attribute name="title">
         <string>Locomotion</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Motor Speed Feedback</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Front Left</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_43">
                    <item>
                     <widget class="QProgressBar" name="locomotion_FrontLeftForwardSpeedBar">
                      <property name="value">
                       <number>24</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QProgressBar" name="locomotion_FrontLeftReverseSpeedBar">
                      <property name="value">
                       <number>24</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="invertedAppearance">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Front Right</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <spacer name="horizontalSpacer_8">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_44">
                    <item>
                     <widget class="QProgressBar" name="locomotion_FrontRightForwardSpeedBar">
                      <property name="value">
                       <number>24</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QProgressBar" name="locomotion_FrontRightReverseSpeedBar">
                      <property name="value">
                       <number>24</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="invertedAppearance">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_5">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Back Left</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item>
                   <spacer name="horizontalSpacer_9">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_45">
                    <item>
                     <widget class="QProgressBar" name="locomotion_BackLeftForwardSpeedBar">
                      <property name="value">
                       <number>24</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QProgressBar" name="locomotion_BackLeftReverseSpeedBar">
                      <property name="value">
                       <number>24</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="invertedAppearance">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_6">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Back Right</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_12">
                  <item>
                   <spacer name="horizontalSpacer_10">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_46">
                    <item>
                     <widget class="QProgressBar" name="locomotion_BackRightForwardSpeedBar">
                      <property name="value">
                       <number>24</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QProgressBar" name="locomotion_BackRightReverseSpeedBar">
                      <property name="value">
                       <number>24</number>
                      </property>
                      <property name="orientation">
                       <enum>Qt::Vertical</enum>
                      </property>
                      <property name="invertedAppearance">
                       <bool>true</bool>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_7">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Control</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_15">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_16"/>
                </item>
                <item>
                 <widget class="QTabWidget" name="tabWidget_2">
                  <property name="currentIndex">
                   <number>2</number>
                  </property>
                  <widget class="QWidget" name="Config1">
                   <attribute name="title">
                    <string>Config1</string>
                   </attribute>
                  </widget>
                  <widget class="QWidget" name="Config2">
                   <attribute name="title">
                    <string>Config2</string>
                   </attribute>
                  </widget>
                  <widget class="QWidget" name="Config3">
                   <attribute name="title">
                    <string>Config3</string>
                   </attribute>
                  </widget>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <item>
                     <widget class="QPushButton" name="locomotion_LeftButton">
                      <property name="text">
                       <string>Left</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_2">
                      <item>
                       <widget class="QPushButton" name="locomotion_UpButton">
                        <property name="text">
                         <string>Up</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="locomotion_DownButton">
                        <property name="text">
                         <string>Down</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                    <item>
                     <widget class="QPushButton" name="locomotion_RightButton">
                      <property name="text">
                       <string>Right</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <widget class="QProgressBar" name="locomotion_ProgressBar">
                    <property name="value">
                     <number>24</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_ExcavationControl">
        <attribute name="title">
         <string>Excavation Control</string>
        </attribute>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>80</y>
           <width>100</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Basket On</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="plainTextEdit_2">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>250</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="plainText">
          <string>Translation</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="plainTextEdit_5">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>330</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="plainText">
          <string>Speed</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="plainTextEdit_6">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>330</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="plainText">
          <string>Duration</string>
         </property>
        </widget>
        <widget class="QSplitter" name="splitter_3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>110</y>
           <width>495</width>
           <height>31</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QSlider" name="horizontalSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="doubleSpinBox">
          <property name="maximum">
           <double>90.000000000000000</double>
          </property>
         </widget>
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="plainText">
           <string>Arm Angle</string>
          </property>
          <property name="backgroundVisible">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_4">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>250</y>
           <width>207</width>
           <height>29</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>RadioButton</string>
          </property>
         </widget>
         <widget class="QSlider" name="horizontalSlider_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>290</y>
           <width>495</width>
           <height>29</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>RadioButton</string>
          </property>
         </widget>
         <widget class="QSlider" name="horizontalSlider_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QSlider" name="horizontalSlider_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Deposition Control</string>
        </attribute>
        <widget class="QSlider" name="horizontalSlider_9">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>60</y>
           <width>160</width>
           <height>29</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="plainTextEdit_7">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>60</y>
           <width>141</width>
           <height>31</height>
          </rect>
         </property>
         <property name="plainText">
          <string>Dump Angel</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_3">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>120</y>
           <width>100</width>
           <height>22</height>
          </rect>
         </property>
         <property name="text">
          <string>Conveyor</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_19">
        <item>
         <widget class="QTextBrowser" name="consoleOutputTextBrowser"/>
        </item>
        <item>
         <widget class="QPushButton" name="emergencyStopButton">
          <property name="text">
           <string>Emergency Stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>919</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
